### PR TITLE
Inject link instead of widget for bots

### PR DESCRIPTION
**Pull request by lemieux**

### CODE REVIEW

**Comment by mspensieri** [comment]
:+1:

**Comment by chris-rudmin** [comment]
A bit late, but I also some css for `.placeholder` which you could remove as well.

**Comment by lemieux** [comment]
Oh good point @chris-rudmin I'll remove it in another PR. Thanks :)

File: src/js/main.js [hunk 1]
```diff
@@ -274,18 +285,6 @@ var SupportKit = Marionette.Object.extend({
...
-                this._placeHolder.render();
-                this._placeHolder.main.show(new ConversationView());
-                this._placeHolder.$el.addClass('placeHolder').appendTo('body');
-            }
-        }).bind(this));
-    },
-
     _renderWidget: function() {
         this._chatController.getWidget().then(_.bind(function(widget) {
             if (this._placeHolder) {
```
Comment (chris-rudmin) [inline src/js/main.js]: @lemieux  Also this can go,

**Pull request merged**